apex-checks/src/test/resources/checks/testMethodInTestClassCorrect.cls

```diff
@@ -0,0 +1,12 @@
+@isTest
+public class SomeTestClass {
+	
+	public static testMethod String aTestMethod() {
+		return 'x';
```

Comment:
according our checks, a test method should contain assertion statements, should not?